client/src/js/components/bhDateInterval.js

```diff
@@ -79,27 +82,42 @@ function bhDateInterval(moment, bhConstants) {
     vm.dateTo = moment().endOf('year').toDate();
   }
 
+ 
+  function ajustDateType(myvar){
```

Comment:
Are you sure this function is necessary?  It looks like it will just do `new Date()` if `myvar` is defined.  what happens if you don't use it?

Reply:
dates are not displayed misfortunatelly